src/includes/gateways/inpage/class-alma-payment-gateway-pay-now.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Alma_Payment_Gateway_Pay_Now.
+ *
+ * @since 5.0.0
+ *
+ * @package Alma_Gateway_For_Woocommerce
+ * @subpackage Alma_Gateway_For_Woocommerce/includes/gateways/inpage
+ * @namespace Alma\Woocommerce\Gateways\InPage
+ */
+
+namespace Alma\Woocommerce\Gateways\Inpage;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use Alma\Woocommerce\Exceptions\Alma_No_Credentials_Exception;
+use Alma\Woocommerce\Gateways\Alma_Payment_Gateway;
+use Alma\Woocommerce\Helpers\Alma_Cart_Helper;
+use Alma\Woocommerce\Helpers\Alma_Constants_Helper;
+
+/**
+ * Alma_Payment_Gateway_Pay_Now
+ */
+class Alma_Payment_Gateway_Pay_Now extends Alma_Payment_Gateway {
+
+	/**
+	 * The cart helper.
+	 *
+	 * @var Alma_Cart_Helper
+	 */
+	protected $cart_helper;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param boolean $check_basics Basic checks.
+	 *
+	 * @throws Alma_No_Credentials_Exception The exception.
+	 */
+	public function __construct( $check_basics = true ) {
+		$this->id                 = $this->get_gateway_id();
+		$this->has_fields         = $this->has_fields();
+		$this->cart_helper        = new Alma_Cart_Helper();
```

Comment:
It's not better to send the instance new Alma_Cart_Helper in the attribute ?

Reply:
Same as before